runtime/commonTest/src/kotlinx/serialization/json/JsonTransformingSerializerTest.kt

```diff
@@ -18,7 +18,7 @@ class JsonTransformingSerializerTest : JsonTestBase() {
     )
 
     object WrappingJsonListSerializer :
-        JsonTransformingSerializer<List<StringData>>(StringData.serializer().list, "WrappingList") {
+        JsonTransformingSerializer<List<StringData>>(serializer(), "WrappingList") {
```

Comment:
Not sure that it is a good idea until `typeOf` fully supported in JS

Reply:
It is a test that works, not sure what did you mean